### DISCUSSION TITLE
Facing issue when deploying web application using Azure static web app

### DISCUSSION THREAD

**Discussion by Zainabohlic** [microsoft/SATechnicalOnboarding#41] [Questions]
Following the guidelines, I am trying to deploy my website using Azure static web app. But I get the notification that
" RepositoryToken is invalid. Provided token has invalid permissions and cannot be used to setup Github Action CI/CD. Admin rights are required for the repository"
<img width="755" alt="problem" src="https://github.com/microsoft/SATechnicalOnboarding/assets/129050037/32184392-aaf9-4bc0-985c-738995ff5376">

**Comment by ManshiPrasad** [comment]
> Following the guidelines, I am trying to deploy my website using Azure static web app. But I get the notification that " RepositoryToken is invalid. Provided token has invalid permissions and cannot be used to setup Github Action CI/CD. Admin rights are required for the repository" <img alt="problem" width="755" src="https://user-images.githubusercontent.com/129050037/246601376-32184392-aaf9-4bc0-985c-738995ff5376.png">

Hey @Zainabohlic did you get a way out? Actually, I got the same error and I'm stuck.

**Reply by ManshiPrasad** [reply]
Hey @Zainabohlic please go through https://github.com/microsoft/SATechnicalOnboarding/discussions/20#discussioncomment-5755127. It has got the solution to the invalid token issue.

**Comment by SalemTracey** [comment]
@Zainabohlic this is resolved.
